css/block.css

```diff
@@ -233,7 +233,7 @@ wb-value[type*=rect] > input{background-image: url(../images/icon/rect.svg);}
 wb-value[type*=string] > input{background-image: url(../images/icon/string.svg);}
 wb-value[type*=geolocation] > input{ background-image: url(../images/icon/geolocation.svg);}
 wb-value[type*=size] > input{ background-image: url(../images/icon/size.svg);}
-wb-value[type*=date] > input{ background-image: url(../images/icon/calendar.svg);}
```

Comment:
Is a px height necessary? Anywhere we can avoid specifying sizes in pixels makes the UI a bit more robust.